src/gui/editors/AutomationEditor.cpp

```diff
@@ -699,12 +694,12 @@ void AutomationEditor::mouseMoveEvent(QMouseEvent * mouseEvent )
 			{
 				if( QApplication::overrideCursor() )
 				{
-	if( QApplication::overrideCursor()->shape() != Qt::SizeAllCursor )
+					if( QApplication::overrideCursor()->shape() != Qt::SizeAllCursor )
```

Comment:
Would you mind to fix spaces with tabs? This seems to be convention on this project.